query/src/main/kotlin/com/linecorp/kotlinjdsl/query/spec/expression/NestedColumnSpec.kt

```diff
@@ -0,0 +1,35 @@
+package com.linecorp.kotlinjdsl.query.spec.expression
+
+import com.linecorp.kotlinjdsl.query.spec.Froms
+import javax.persistence.criteria.*
+
+private fun <T> ColumnSpec<T>.outerPath(froms: Froms): Path<T> = froms[entity].get(path)
+
+data class NestedColumnSpec<T>(
```

Comment:
> Hello NestColumnSpec is subordinate to ColumnSpec so which one is better
>
> - make NestColumnSpec.kt file
> - move NestColumnSpec class to ColumnSpec.kt file
>
> if you give me feedback I will fix that one

I don't think NestCoulmnSpec is subordinate ColumnSpec. This is because ColumnSpec is only used to express NestColumnSpec and NestColumnSpec has a separate role.

I agree with the code you wrote.